### DISCUSSION TITLE
No more than 500 characters per params value error

### DISCUSSION THREAD

**Discussion by matsuobasho** [mlflow/mlflow#12904] [General]
I'm finetuning a T5 encoder-decoder model using AzureML.
I get an error in the part of my code where I initialize MLFlow.

```python
    with mlflow.start_run():
        mlflow.transformers.autolog(log_models=False)
        trainer = Seq2SeqTrainer(model,
                                 training_args,
                                 train_dataset=data_tokenized["train"],
                                 eval_dataset=data_tokenized["val"],
                                 data_collator=data_collator,
                                 compute_metrics=compute_metrics_partial,
                                 tokenizer=tokenizer)
```

I looked around and found some issues on the `transformers` repo.
I did attempt to add the custom class mentioned [here](https://github.com/huggingface/transformers/issues/8967), but it didn't eliminate the error.

```python
class DummyClass:
    def to_dict(self):
        return {}
    
model.config.encoder = DummyClass()
model.config.decoder = DummyClass() 
```

**Comment by B-Step62** [comment]
@matsuobasho We have upgraded the parameter value length limit from 500 to 6000in MLflow 2.8.0. Please upgrade the MLflow version and run `mlflow db upgrade` command to run the migration.
